internal/libooniengine/ooniengine.h

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#ifndef OONI_ENGINE_H
+#define OONI_ENGINE_H
```

Comment:
I think it would be better to name this file just `engine.h`. If we assume it would be installed inside `<ooniengine/engine.h>`, the include guards need to become `OONIENGINE_ENGINE_H`.